src/EditorFeatures/Core.Wpf/NavigateTo/NavigateToItemDisplay.cs

```diff
@@ -7,6 +7,7 @@
 using System.Collections.Generic;
 using System.Collections.ObjectModel;
 using System.Drawing;
+using System.Runtime.CompilerServices;
```

Comment:
📝 All changes to this file will be reverted when compatible editor packages are available. Until then, this assembly is forced into an unusable state.